packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -352,7 +351,12 @@ export default function (
   const suites = getFixtures(fixturesLoc);
 
   for (const testSuite of suites) {
-    if (includes(suiteOpts.ignoreSuites, testSuite.title)) continue;
+    if (
+      suiteOpts.ignoreSuites &&
+      suiteOpts.ignoreSuites.includes(testSuite.title)
```

Comment:
```suggestion
      suiteOpts.ignoreSuites?.includes(testSuite.title)
```

Reply:
Thanks!  I *think* this also means we can get rid of the `suiteOpts.ignoreSuites` condition on the previous line?  (I haven't used the optional chaining operator much).

I've pushed a change that applies that simplification here and also in the `ignoreTasks` block a few lines further down.